### DISCUSSION TITLE
希望可以推出一个ARM32的版本

### DISCUSSION THREAD

**Discussion by a725422188** [ywmoyue/biliuwp-lite#390] [General]
### 问题描述

我知道这个需求不是很热门，因为基本上只有window phone系统有这个需求，非常喜欢这个系统，而这个系统普遍停留在win10m 1703和1709， 设备也很老，基本上都是32位的。B站官方也基本上在上个月彻底停止了这些系统UWP的B站登录和直播观看。 现在只能看一些360P的视频了。表述有些乱，但是如果不麻烦的话，希望大佬可以出一个，不胜感激。

### 描述解决方案

如果不麻烦希望可以编译一个ARM32的版本，如果麻烦就算了，毕竟小众。感谢。

### 代码贡献

否

**Comment by ywmoyue** [comment]
有几个问题
1. win10m能安装msix应用包吗
2. 这个应用最低要求操作系统版本已经升级到1903了
3. 这个应用是桌面端应用，没有针对小屏幕设备做特殊优化
